### PR TITLE
Add caption vvaths

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @SoDaCoDa
- [ ] @bvetter95
- [ ] @daver33
- [ ] @enthusiast
- [ ] @gzone404
- [ ] @hannalyst
- [ ] @industrylol
- [ ] @jpostenwapa
- [ ] @larrybaugh
- [ ] @ndhillon101
- [ ] @reservoirdoll
- [ ] @sevans-wapa
- [ ] @tehowardwapa
- [ ] @vvaths
- [ ] @BouchardWAPA
- [ ] @lchu456

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/03-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Magic Caption!
```
Comment (vyas000) [inline _slides/03-vvaths.md]:
```suggestion
Awesome Magic Caption!!!
```

File: _slides/03-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![poptocat](https://octodex.github.com/images/poptocat.png)
```
Comment (vyas000) [inline _slides/03-vvaths.md:8]: nice octocat

**Review by vyas000 — CHANGES_REQUESTED** [review]
looks good. make the change requested!

File: _slides/03-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/03-vvaths.md:8]: thanks

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**